### DISCUSSION TITLE
Software engineer

### DISCUSSION THREAD

**Discussion by Koosha-Owji** [WeAreMahsaAmini/Core#467] [Introductions]
I'm a computer science student from melbourne, I have a lot of free time on my hand and i'm looking to help! i have worked with a few full stack projects.

**Comment by Pink3lephant** [comment]
Hi @Koosha-Owji 

Nice to hear from you. We are working on this project and we need help with our Frontend development [Nextjs, Strapi] (it's in progress, but we need more resources), can you please have a look and let me know if you're interested in helping out? 

🐜  Current website: [Antyfilter.com](https://Antyfilter.com)
👨‍💻 [Github](https://github.com/Antyfilter)


You can email me at `freeinternettech` [at] gmail.

**Comment by tcf-admin** [comment]
Hi @Koosha-Owji, 
check out forDem's introduction here: https://github.com/WeAreMahsaAmini/Core/discussions/495
Hope you find it interesting and we can have you next to us.
